src/endpoints.js

```diff
@@ -1,10 +1,17 @@
 import environment from './environment'
 
 import { isLocalOrUnknownNetwork, getNetworkType } from './lib/web3-utils'
+import { getNetworkConfig } from './networks'
+import {
+  getDefaultEthNode,
+  getIpfsGateway,
+  getSubgraphHttpEndpoint,
+  getSubgraphWsEndpoint,
+} from './local-settings'
 
 const CHAIN_ID = environment('CHAIN_ID')
 const COURT_SERVER_NAME = environment('COURT_SERVER_NAME')
-const SUBGRAPH_NAME = environment('SUBGRAPH_NAME')
+// const SUBGRAPH_NAME = environment('SUBGRAPH_NAME')
```

Comment:
Should we remove this? The logic is now in `networks.js`?

Reply:
yep definitely